src/AvPortalResource.php

```diff
@@ -78,17 +79,17 @@ public function getTitle(string $langcode = 'EN'): ?string {
 
     $titles = $this->data['titles_json'];
     if (isset($titles[$langcode])) {
-      return $titles[$langcode];
+      return Xss::filter($titles[$langcode], []);
```

Comment:
`Xss::filter` is generally used in conjunction with `#markup` elements and alike. Since we have autoescaping, this leads to double encoded html entities (`&` becomes `&amp;amp;` instead of `&amp;`).
Let's use strip_tags to just remove the markup.